docs/reference/releases-servicing.md

```diff
@@ -7,11 +7,11 @@ nav_order: 1
 
 # ONNX Runtime releases
 
-The current ONNX Runtime release is [1.18](https://github.com/microsoft/onnxruntime/releases/tag/v1.18.0).
+The current ONNX Runtime release is [1.19](https://github.com/microsoft/onnxruntime/releases/tag/v1.19.0).
 
-The next release is ONNX Runtime release 1.19.
+The next release is ONNX Runtime release 1.20.
```

Comment:
I would suggest to add a link instead, and not mention the exact version number (otherwise, it could be out of sync like currently). For example current release is 1.9.2, not 1.9.0.

Current release: https://github.com/microsoft/onnxruntime/releases
Next release: https://onnxruntime.ai/roadmap

Reply:
Done!